r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -80,6 +80,25 @@ test_that("extract month from timestamp", {
       collect(),
     test_df
   )
+
+  skip_on_os("windows") # https://issues.apache.org/jira/browse/ARROW-13168
+
+  compare_dplyr_binding(
+    .input %>%
+      # R returns ordered factor whereas Arrow returns character
```

Comment:
I wonder if this is a fact we should warn about?

Reply:
Yeah, was wondering this too.  Do you think it would be better as a warning message or just in the docs somewhere?

Reply:
Docs only is probably just fine

Reply:
I have added a comment here to mention it somewhere in the vignettes: https://issues.apache.org/jira/browse/ARROW-12137
I have also opened a ticket here to mention things like this in the cookbook: https://github.com/apache/arrow-cookbook/issues/105